templates/endpoints_v2/README.ipynb

```diff
@@ -20,18 +20,11 @@
     "\n",
     "## Step 1 - Run the model locally in the Workspace\n",
     "\n",
-    "We provide a starter command to run Llama-2 and Mistral-family models via Ray Serve. Specify the model ID, GPU type, and tensor parallelism with the command arguments. You can also follow the [guide](examples/CustomModels.ipynb) to bring your own models.\n",
```

Comment:
We should simplify CustomModels, Lora and function calling instructions now that we have this